### PR TITLE
Adding AussieFelatioQueens.com scraper

### PR DESCRIPTION
**Pull request by Tweeticoats**
Adding AussieFelatioQueens.com scraper

### CODE REVIEW

File: scrapers/AussieFelatioQueens.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Aussie Fellatio Queens
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://aussiefellatioqueens.com/
```
Comment (halorrr) [inline scrapers/AussieFelatioQueens.yml:5]: Can drop the protocol so it is just `aussiefellatioqueens.com/`

File: scrapers/AussieFelatioQueens.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      Details:
+        selector: //h3[@class='text-video-desc text-color-5 full-desc']/span/text()
+      Tags:
+        Name:
+          selector: //div[@class='tags content-info']/h4/span/a/span/text()
+driver:
+  useCDP: true
+debug:
+  printHTML: true
+# Last Updated August 26, 2020
```
Comment (halorrr) [inline scrapers/AussieFelatioQueens.yml:24]: Need to remove debug and update the date

File: scrapers/AussieFelatioQueens.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h2[@class='video-title fleft text-color-3']/span[2]/text()
+      Performers:
+        Name: 
+          selector: //span/span/a/span/text()
+      Details:
+        selector: //h3[@class='text-video-desc text-color-5 full-desc']/span/text()
```
Comment (halorrr) [inline scrapers/AussieFelatioQueens.yml:16]: This selector didn't work on certain scenes like `https://aussiefellatioqueens.com/scene/8074313/swallow-booth-rubi-valentine-1st-visit-cumshot-compilation`

This seems to solve that issue:
```
selector: //h3[contains(@class,'text-video-desc') and not(contains(@class,'short-desc'))]/span/text()
```

**Review by halorrr — COMMENTED** [review]
Can also throw in the studio with 

```YAML
Studio:
  Name:
    fixed: Aussie Felatio Queens
```

**Comment by halorrr** [comment]
Based on my comments this would be the updated scraper:

```YAML
name: Aussie Fellatio Queens
sceneByURL:
  - action: scrapeXPath
    url:
      - aussiefellatioqueens.com/
    scraper: sceneScraper
xPathScrapers:
  sceneScraper:
    scene:
      Title:
        selector: //h2[@class='video-title fleft text-color-3']/span[2]/text()
      Performers:
        Name:
          selector: //span/span/a/span/text()
      Details:
        selector: selector: //h3[contains(@class,'text-video-desc') and not(contains(@class,'short-desc'))]/span/text()
      Tags:
        Name:
          selector: //div[@class='tags content-info']/h4/span/a/span/text()
driver:
  useCDP: true

# Last Updated October 23, 2020
```

**Comment by bnkai** [comment]
updated with suggested changes, added studio and merged.

**Pull request merged**